azavea/raster-foundry
# Add approval_required and start_time fields to AoisToProjects model

## Overview
Add field to store whether an AOI should automatically add scenes to a project
without approval
Add a field to store the AOI start time

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Notes

Logic for actually using this field will be in a future PR when the feature is more fleshed out

## Testing Instructions
* Run the migration
* Verify that `./scripts/test` runs successfully (or just wait for commit build)
* Verify in the DB that the new columns exist on `aois_to_projects`

Closes #1765 
Closes #1764
